packages/wallets/src/features/cashier/modules/Transactions/Transactions.tsx

```diff
@@ -48,10 +54,10 @@ const Transactions = () => {
     );
 
     useEffect(() => {
-        if (!wallet?.currency_config?.is_crypto && isPendingActive) {
+        if (!isLoading && !wallet?.currency_config?.is_crypto && isPendingActive) {
```

Comment:
interesting case: currency_config is typically being loaded and added to wallets object only AFTER wallets are available... meaning that initial value of is_crypo is gonna be evaluated to false and trigger setIsPendingActive(false) even when it not supposed to, 

so added the extra !isLoading (for currency_config) to ensure that wallets data have the currency_config attached before we use wallet.currency_config.is_crypto,

Reply:
probably useActiveWalletAccount should ensure that isLoading is false until all the dependand hooks wont return all the data (e.g. wait until currency_config is loaded within useActiveWalletAccount hook)